zulipterminal/core.py

```diff
@@ -682,8 +692,18 @@ def _raise_exception(self, *args: Any, **kwargs: Any) -> Literal[True]:
                     + "\n\n"
                     + "Details of the exception can be found in "
                     + exception_logfile
+                    + "\n\n"
+                    + "Press 'c' to copy traceback to clipboard."
+                )
+                self.exception_popup_with_message(
+                    message, traceback=full_traceback, width=80
+                )
+                self.report_error(
```

Comment:
I'd definitely suggest a distinct way of reporting to the user could be in a separate commit - whether squashed or dropped.

If the intention is to emphasize the error and the available hotkey, then I'd suggest we use a different style in the popup instead.

The footer is generally used for temporary messages, rather than alerting users to hotkeys or events - and we should have the popup to indicate that? Or is this to handle a different case? It would also seem odd to duplicate it, if that is what is happening here?

(In addition, if we intend to use the footer for available hotkeys more broadly, it would seem strange to have this show a hotkey, but only temporarily)

Reply:
I put it in a separate commit and took a different approach to setting the footer text, resetting it only when exiting the popup.